recipes/r-quilt/build.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -xe
+
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 
-cp $SRC_DIR/QUILT.R $PREFIX/bin/
+cp -rf $SRC_DIR/QUILT.R $PREFIX/bin/
```

Comment:
This looks dangerous, why is it needed?

Reply:
Yes, `r-quilt` is not only a pure r package, but also includes the CLI `QUILT.R` dependent on `r-optparse`. Normally, the users of QUILT run `QUILT.R` extensively on large sequences analyses instead of loading `library(QUILT)` interactively in R.

Reply:
Do you suggest removing the option `-rf`?